packages/embedder/importHistoricalRetrospectiveDiscussionTopic.ts

```diff
@@ -0,0 +1,36 @@
+import getKysely from 'parabol-server/postgres/getKysely'
+import {Logger} from 'parabol-server/utils/Logger'
+import RedisInstance from 'parabol-server/utils/RedisInstance'
+import {addEmbeddingsMetadataForRetrospectiveDiscussionTopic} from './addEmbeddingsMetadataForRet'
+
+export const importHistoricalRetrospectiveDiscussionTopic = async (redis: RedisInstance) => {
+  const pg = getKysely()
+  const isEarliestMetadataImported = await pg
```

Comment:
@jordanh your strategy was best! i'm doing this here on startup. doing this in a migration would require that the embedder service be up at the time that the migration ran, which we can't guarantee